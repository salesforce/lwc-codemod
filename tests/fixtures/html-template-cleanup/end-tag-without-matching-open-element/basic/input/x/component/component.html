<template>
    <span>hello</span>
</template>
</template>