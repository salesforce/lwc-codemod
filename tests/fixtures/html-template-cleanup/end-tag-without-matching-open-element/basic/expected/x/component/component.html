<template>
    <span>hello</span>
</template>
